build/yaml/crd/nsx.vmware.com_subnets.yaml

```diff
@@ -0,0 +1,106 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: subnets.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: Subnet
+    listKind: SubnetList
+    plural: subnets
+    singular: subnet
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Subnet is the Schema for the subnets API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubnetSpec defines the desired state of Subnet
+            properties:
+              accessMode:
+                default: private
+                description: Access mode of subnet, accessible only from within VPC
+                  or from outside VPC. Defaults to private.
+                enum:
+                - private
+                - public
+                - isolated
+                type: string
+              ipAddresses:
+                description: Subnet CIDR.
+                items:
+                  type: string
+                maxItems: 2
```

Comment:
Why maxItems is 2, for v4 and v6?

Reply:
Not sure about this, @lxiaopei do you know about it?

Reply:
yes, for ipv4 and ipv6.